Tools/cases_generator/test_generator.py

```diff
@@ -333,20 +333,21 @@ def test_super_instruction():
 
 def test_macro_instruction():
     input = """
-        inst(OP1, (counter/1, arg --)) {
-            op1();
+        inst(OP1, (counter/1, arg1 -- interim)) {
+            interim = op1(arg1);
         }
-        op(OP2, (extra/2, arg --)) {
-            op2();
+        op(OP2, (extra/2, arg2, interim -- res)) {
+            res = op2(arg2, interim);
         }
         macro(OP) = OP1 + cache/2 + OP2;
     """
     output = """
         TARGET(OP1) {
-            PyObject *arg = PEEK(1);
+            PyObject *arg1 = PEEK(1);
+            PyObject *interim;
```

Comment:
Note that if you run the tests it'll fail here. I accidentally hit return in the middle of this line and didn't re-run these tests before pushing. As these tests aren't run automatically yet this didn't trigger any CI failures. :-(

I'll fix it in my next change to this file, but I figured I'd warn you.

Reply:
We also need to update the makefile. Currently when I change the opcode_metadata.h file I need to touch compile.c. I had a look but wasn't sure what to change.